### ISSUE TITLE
Add `other_attributes` to company-for-nesting.json

### ISSUE THREAD

**Issue by jpmckinney** [openc/openc-schema#39]
France stores the `registrar` for the company, i.e. the official responsible for adding the company to the register. I have nowhere to put this - unless I dissociate it from the entity and store it in the notice's `other_attributes`, but that dissociation makes it unlikely anyone will see it.

**Comment by CountCulture** [comment]
I'm agnostic about this. Think it prob makes sense for it to be in the
company other_attributes -- however this isn't yet added to the company
(for both good/less good reasons). So practically it doesn't make much
difference yet

On 11 January 2016 at 21:05, James McKinney <notifications@github.com>
wrote:

> France stores the registrar for the company, i.e. the official
> responsible for adding the company to the register. I have nowhere to put
> this - unless I dissociate it from the entity and store it in the notice's
> other_attributes, but that dissociation makes it unlikely anyone will see
> it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openc/openc-schema/issues/39>.
>



-- 
-------------------------------------------------------
OpenCorporates :: The Open Database of the Corporate World
http://opencorporates.com
OpenlyLocal :: Making Local Government More Transparent
http://openlylocal.com
Blog: http://countculture.wordpress.com
Twitter: http://twitter.com/CountCulture

**Comment by jpmckinney** [comment]
Same goes for `activite` (activity, e.g. "pose de joints sur plaque de plâtre"). Since there may be multiple subject entities within a notice, I'd have to copy the entire entity into the base `other_attributes` just to add this on field, as other ways there's no way to figure out which entity goes with which activity.

**Comment by CountCulture** [comment]
Couple of example notices?

**Comment by jpmckinney** [comment]
Here's a good example: http://www.bodacc.fr/annonce/detail/BXA14358000590H

**Comment by jpmckinney** [comment]
Also needed for `capital`, e.g. http://www.bodacc.fr/annonce/detail/BXA14358001500V

**Comment by CountCulture** [comment]
Can you clarify exactly what the problem is with these?

**Comment by jpmckinney** [comment]
There's just nowhere to store them in a sensible way.

**Comment by CountCulture** [comment]
No, I mean I don't understand what the specific problem is regarding those
two notices. Can you annotate them or explain so that I (or anyone else)
can understand the problem at a glance.
It's quite common for gazette notices to have more than on company
referenced


On 29 January 2016 at 23:48, James McKinney <notifications@github.com>
wrote:

> There's just nowhere to store them in a sensible way.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openc/openc-schema/issues/39#issuecomment-177021974>.
>



-- 
-------------------------------------------------------
OpenCorporates :: The Open Database of the Corporate World
http://opencorporates.com
OpenlyLocal :: Making Local Government More Transparent
http://openlylocal.com
Blog: http://countculture.wordpress.com
Twitter: http://twitter.com/CountCulture
